triton-server/src/download/schema.sql

```diff
@@ -46,4 +46,11 @@ create table contacts (
     type       text check(type IN ('SFTP', 'GLOBUS')) not null,
 
    unique(project_id, type)
-);
\ No newline at end of file
+);
+
+create table constants (
```

Comment:
We should include in this file the actual values to be inserted in the table... This should be run as part of initializing the server.

Reply:
Eh I believe we should include all ;)
Even failed staging will take up space anyways.